templates/authenticate-deployment.yaml

```diff
@@ -85,11 +83,13 @@ spec:
               key: idp-client-secret
         - name: IDP_PROVIDER_URL
           value: {{ .Values.authenticate.idp.url }}
+{{- if .Values.authenticate.idp.serviceAccount }}
```

Comment:
@desimone this prevents the use of a service account when rolling my own secret containing the service account.

As a workaround, I can just set `authenticate.idp.serviceAccount` to `true` though. I'm not sure how to fix this so we may need to leave it like this but simply document the fact that if needed, it can be set to `true` when using `existingSecret`.

Reply:
Lets track this in a new issue, #7